docs/journal/01-cold-storage.md

```diff
@@ -76,14 +92,13 @@ In this configuration, devices post event data to a Service Bus topic. Worker ro
 
 - **Throughput.** A worker role has less overhead than a web role (*NEED SOMEONE TO VERIFY THIS, AND MORE DETAIL ON WHY*)
 
-- **Reliability.** The AMQP protocol supports reliability guarantees, helping to ensure that event data is not lost when sent to a topic. Additionally, a subscription can be transactional; if a worker role fails while processing the data for an event this data is not lost but is instead returned to the Service Bus topic from whence it can be retrieved again.
+- **Reliability.** The AMQP protocol supports reliability guarantees, helping to ensure that event data is not lost when sent to a topic. Additionally, a subscription can be transactional; if a worker role fails while processing an event, the event is returned to the Service Bus topic and can be retrieved again.
 
-- **Scalability.** Different worker roles could be created to handle different types of event processing (copying to cold storage, performing warm analysis and aggregation, raising alerts).
+- **Scalability.** Different worker roles can handle different types of event processing (copying to cold storage, performing warm analysis and aggregation, raising alerts).
```

Comment:
I think that _routing to different workers by type_ is actually a special case. One that we haven't actually pursued. However, being able increase the number of concurrent workers (even w/o differentiating by event type) is very important for scale. In addition, we can scale the number of workers _processing_ indepent of the actual ingestion.